crates/cxx-qt-gen/src/generator/cpp/fragment.rs

```diff
@@ -47,4 +60,11 @@ mod tests {
             }
         )
     }
+
+    #[test]
+    fn test_into_string() {
+        let source = CppFragment::Source(String::from("SOURCE"));
+        let str_repr: String = source.into();
+        assert_eq!(str_repr.as_str(), "SOURCE")
+    }
```

Comment:
this could be removed too?

Reply:
Needed for coverage of the into() trait I believe